dom-fetch-client/src/main/scala/org/http4s/dom/FetchClient.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2021 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+package dom
+
+import cats.effect.Async
+import cats.effect.Resource
+import cats.effect.syntax.all._
+import cats.syntax.all._
+import fs2.Stream
+import org.http4s.client.Client
+import org.scalajs.dom.crypto._
+import org.scalajs.dom.experimental.Fetch
+import org.scalajs.dom.experimental.Headers
+import org.scalajs.dom.experimental.HttpMethod
+import org.scalajs.dom.experimental.ReadableStream
+import org.scalajs.dom.experimental.RequestInit
```

Comment:
It's not 1.0 yet, but I wonder what we're getting into as we move toward 1.0 on top of something experimental.  We'll probably have to flag it loudly in docs and release notes?

Reply:
This is no longer experimental, just binary compatibility. The `scala-js-dom` library is releasing a 2.0 soon with a proper Scala 3 artifact and these moved out of experimental. See https://github.com/scala-js/scala-js-dom/issues/451#issuecomment-891141873